modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/SequenceSearch.scala

```diff
@@ -58,9 +59,8 @@ object SequenceSearchResultsBody {
               <.td(u.instrument),
               <.td(
                 ^.cls := "collapsing",
-                <.button(
-                  ^.cls := "circular ui icon button",
-                  Icon(Icon.Props("plus", onClick = onAdding(p.searchResults, u)))
+                Button(Button.Props(icon = true, circular = true, onClick = onAdding(p.searchResults, u)),
+                  Icon(Icon.Props("plus"))
```

Comment:
I didn't totally get your comment, you were talking about having the Icon twice. The `Button` class could be improved to take an `Icon` in its props and assume the `icon` flag if that's the case